### PR TITLE
added task with *

### PR DESCRIPTION
**Pull request by DavidTereshchenko**

### CODE REVIEW

**Comment by DavidTereshchenko** [comment]
some fixed

**Comment by DavidTereshchenko** [comment]
always forget to change the commit name, sorry

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html [hunk 1]
```diff
@@ -32,12 +32,27 @@
...
+                        </div>
+                        <div class="result__block">
+                            <p class="resul__number result"></p>
+                        </div>
                     </div>
                 </div>
             </main>
         </div>
         <script src="homework.js"></script>
+        <script src="calculator.js"></script>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html:56]: Are you sure you need both of these scripts?
On each file you declared a variable with the equal name 'operator' and ' btn', so your logic doesn't work as you have an error in the console, please fix it:
Screenshot 2023-11-07 at 01 59 12

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html [hunk 2]
```diff
@@ -32,12 +32,27 @@
                     <div class="return">
                         <button type="button" class="button">Calculation</button>
                     </div>
-                    <div class="re">
-                        <p class="resul__number result"></p>
+                    <div class="count-block">
+                        <div class="visual-counting">
+                            <div class="number__one">
+                                <h2 class="number__1"></h2>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html:38]: It is not a title. `h2` - is the semantic tag for a title, important for the search machine to analyze your page. It is `<span>`

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html [hunk 3]
```diff
@@ -32,12 +32,27 @@
...
+                            </div>
+                            <div class="operator__visual">
+                                <h2 class="oper"></h2>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html:41]: It is not a title. Title - is the semantic tag, important for the search machine to analyze your page. It is `<span>`.

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html [hunk 4]
```diff
@@ -32,12 +32,27 @@
...
+                            </div>
+                            <div class="number__two">
+                                <h2 class="number__2"></h2>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html:44]: The same mistake with `h2`

File: homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html [hunk 5]
```diff
@@ -32,12 +32,27 @@
...
+                            </div>
+                            <div class="equals"></div>
```
Comment (anna-vrana-moc) [inline homeworks/david.tereshchenko_DavidTereshchenko/5-logic/index.html:46]: It is also `<span>`. You can write `<span class="equals">=</span> ` and toggle `display: block` for `<div class="count-block">`

**Comment by anna-vrana-moc** [comment]
Hi @DavidTereshchenko!
 According to the extra task requirements, you need to add one more page, a "calculator" page, to your project and include a link to it in the header. The page should have a header and a footer, similar to the other pages in the project. Additionally, the styles should match the project's overall design (fonts, colors), and you can optionally add a visualization of the result in the form of cards.
 So, for now, I canʼt raise your grade because you did not fully meet the task requirements 🥲

**Comment by anna-vrana-moc** [comment]
@DavidTereshchenko avoid using spaces in naming (you have the folder 'default selectors'). Different tools can interpret it unexpectable so you can get a broken path. Ex. from WebStorm:
By the way, it seems you should fix the styles
Screenshot 2023-11-08 at 00 11 02

Screenshot 2023-11-08 at 00 12 07

**Comment by anna-vrana-moc** [comment]
I believe you can fix the issue with styles and get your star 💫

**Comment by DavidTereshchenko** [comment]
> I believe you can fix the issue with styles and get your star 💫

wow, i don't have this problem

**Comment by DavidTereshchenko** [comment]
> I believe you can fix the issue with styles and get your star 💫

i don't added for commit someone file

**Pull request merged**